twill-api/src/main/java/org/apache/twill/api/Configs.java

```diff
@@ -34,6 +36,11 @@
     public static final String JAVA_RESERVED_MEMORY_MB = "twill.java.reserved.memory.mb";
 
     /**
+     * Configurable ratio between Heap and Reserved Memory
+     */
+    public static final String HEAP_RESERVED_MIN_RATIO_CONFIG = "twill.java.reserved.memory.ratio";
```

Comment:
Also, I think the config key should be `twill.java.min.heap.memory.ratio`, as it is the minimum ratio of heap memory.